languages/es_ES.po

```diff
@@ -149,11 +149,11 @@ msgstr "bosques"
 
 #: Mongabay-News-Wordpress-Theme/components/lib/topics-section.php:59
 msgid "indigenous-peoples"
-msgstr "pueblos-indigenas"
+msgstr "pueblos-indígenas"
```

Comment:
Lower case strings are actually url safe permalinks so it should remain 'pueblos-indigenas'. Correct topic name is fetched from data base using this slug.